opensearch/src/main/java/org/opensearch/sql/opensearch/storage/scan/CalciteOpenSearchIndexScan.java

```diff
@@ -118,17 +138,18 @@ public Enumerator<Object> enumerator() {
     };
   }
 
-  public boolean pushDownFilter(Filter filter) {
+  public CalciteOpenSearchIndexScan pushDownFilter(Filter filter) {
```

Comment:
why change the return type? seems the return value is never in use besides the null-checker in `OpenSearchFilterIndexScanRule`